### DISCUSSION TITLE
助理Pro的SERPAPI_API怎么启用？

### DISCUSSION THREAD

**Discussion by LeXwDeX** [GaiZhenbiao/ChuanhuChatGPT#812] [Q&A] [answered]
如题，填写了KEY之后，没在界面找到地方。目前还是默认的DUCKDUCKGO。

**Comment by Keldos-Li** [comment, marked as answer]
Hi，在模型下拉列表中选择“川虎助理Pro”，此时模型在思维链中将会调用Google进行检索。
<img width="345" alt="image" src="https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/23137268/2aa62bb5-0d7e-46f6-80ba-e883817a8e38">

**Reply by LeXwDeX** [reply]
各种KEY都填写好了，但Pro模式仍然用的DUCKDUCKGO

**Comment by GaiZhenbiao** [comment]
“Pro模式仍然用的DUCKDUCKGO”的症状是什么呢？Pro 模式一定会使用 Google/SerpAPI 的，因为根本没有使用 DDG 的功能。

**Reply by LeXwDeX** [reply]
我重启了下好了，不知道是哪里没高队。
判断使用Duckduckgo的逻辑就是查询出来的结果。
